src/components/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Comment:
you don't use proptypes, your var must have types use like this:
Sum.propTypes = {
  a: PropTypes.number,
  b: PropTypes.number,
};